scripts/pre_commit_linter.py

```diff
@@ -463,8 +470,239 @@ def _redirect_stdout(new_target):
         sys.stdout = old_target
 
 
+def _get_all_files():
+    """This function is used to get the list of all files for linting and
+    pattern checks.
+
+    Returns:
+        list(str, bool). The list of files for linting and pattern checks and
+            bool for mode.
+    """
+    eslintignore_path = os.path.join(os.getcwd(), '.eslintignore')
+    parsed_args = _PARSER.parse_args()
+    # Default mode is non-verbose mode, if arguments contains -v flag it will
+    # be made True, which will represent verbose mode.
+    mode = False
```

Comment:
Also, here and elsewhere, go with `verbose_mode_enabled`. Clearer, and obviously "boolean" in nature.

Reply:
Done